index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+  <meta charset="utf-8">
+
+  <link rel="stylesheet" href="style.css">
+
+  <script src="script.js"></script>
+
+  <title>Tic Tac Toe</title>
+
+</head>
+
+<body onload="startGame();">
+
+  <header id="header">
+    Let's play Tic Tac Toe!!
+  </header>
+
+  <div id="message">
+  </div>
+
+  <table>
+    <tr>
+      <td id="box1" onclick="nextMove(this);"></td>
```

Comment:
How could we dry up this code? Maybe try adding classes to each `box`? Then hooking up a click listener to that class, on which we run the `nextMove` function?